### PR TITLE
Remove multiple branches when working with remotes

### LINKED ISSUE #3: Review Remote repositories episode
Review by reframing as single-branch workflows. i.e. remove references to "feature branch", clarify the basic concept of branches but stick with one "main" branch for this course.

### PR DESCRIPTION
**Pull request by dalonsoa**
Reviews episode on `Remote Repositories`, removing any mention of working with multiple branches. While a generic `branch` name is used throughout the episode, it is made clear at the beginning that we are talking about the `main` branch and that these concepts are all applicable to other branches.

A mention is done on merging and conflicts, but deferred to the intermediate course.

Close #3

### CODE REVIEW

File: _episodes/l2-02-remote_repositories.md [hunk 1]
```diff
@@ -41,6 +37,8 @@ keypoints:
 - A local branch synchronised with a remote one is said to be **tracking** that
   remote branch.
 - The remote branch being tracked is called the **upstream branch**.
+- As mentioned, **we will be working in this course with the main branch only**, but the
```
Comment (dandavies99) [inline _episodes/l2-02-remote_repositories.md:40]: Where is this previously mentioned? In a previous episode?

File: _episodes/l2-02-remote_repositories.md [hunk 2]
```diff
@@ -88,26 +89,25 @@ local one out of it or the other way around, the steps are different.
 > This involves the git command **clone**. Let's create a local copy of the
 > `example` repository you created remotely in the last episode.
 >
-> - On GitHub, the press down arrow in the far top right and choose "Your
->   repositories" from the drop-down menu.
+> - On GitHub, press down arrow in the far top right, next to your picture, and choose
```
Comment (dandavies99) [inline _episodes/l2-02-remote_repositories.md:92]:
```suggestion
> - On GitHub, press the down arrow in the far top right, next to your picture, and choose
```

File: _episodes/l2-02-remote_repositories.md [hunk 3]
```diff
@@ -143,34 +135,40 @@ local one out of it or the other way around, the steps are different.
...
- checkout [branch_name]` without the need of creating the branch locally and
-  then pulling the remote. 
+  unstaged changes - so, all changes committed.
 
 ![Git collaborative]({{ site.baseurl }}/fig/pull.png "Pull remote changes")
 {:class="img-responsive"}
 
+> ## Conflicts
+>
+> If the local and upstream branches have diverged - for example if you editted
```
Comment (dandavies99) [inline _episodes/l2-02-remote_repositories.md:160]:
```suggestion
> If the local and upstream branches have diverged - for example if you edited
```

**Review by dandavies99 — CHANGES_REQUESTED** [review]
Nice - just a couple of minor suggestions.

File: _episodes/l2-02-remote_repositories.md (hunk 1 above)
Comment (dalonsoa) [reply _episodes/l2-02-remote_repositories.md:40]: Well... I should have said that this should be mentioned in the previous episode, commits and history, so a small task for @dc2917 .

**Review by dandavies99 — APPROVED** [review]

**Review by dc2917 — APPROVED** [review]

**Pull request merged**